index.html

```diff
@@ -37,11 +37,41 @@ <h1 class="logo-heading">Fun Bus</h1>
   <div class="container">
     <div class="row">
         <div class="col-sm-12">
-            <img src="img/fun-bus.jpg" class="img-fluid" alt="Responsive image">
+          <div id="carouselExampleIndicators" class="carousel slide" data-ride="carousel">
```

Comment:
I love how you implemented the carousel at the top, it's a feature that I would definitely like on a site!